cpp/src/interop/from_arrow_device.cu

```diff
@@ -379,11 +336,25 @@ dispatch_tuple_t get_column(ArrowSchemaView* schema,
 
 }  // namespace
 
-unique_table_view_t from_arrow_device(ArrowSchemaView* schema,
+unique_table_view_t from_arrow_device(ArrowSchema const* schema,
```

Comment:
This was because of @davidwendt's request that we shift the view init into the function impl inside the `detail` for other callers. see https://github.com/rapidsai/cudf/pull/15645#discussion_r1602115746

Reply:
Right. The view init was happening in the public API definition and this just moves it to the `detail` implementation instead I believe.